Updated credtis

Tom-van-Woudenberg opened the pull request:
See https://mude.citg.tudelft.nl/book/credits-2025/credits.html

Tom-van-Woudenberg committed:
Updated credtis

Tom-van-Woudenberg committed:
Update credits.md

Tom-van-Woudenberg committed:
Update credits.md

Tom-van-Woudenberg committed:
update mude2025 to most recent

Tom-van-Woudenberg committed:
Further updates

Added edition information to the 2024 MUDE textbook BibTeX entry and updated URLs for the most recent version. Improved citation instructions in credits.md, clarifying how to reference specific chapters or pages and emphasizing the importance of including URLs and version numbers for accurate referencing.

Tom-van-Woudenberg committed:
final

Tom-van-Woudenberg committed:
remove first DOI

Tom-van-Woudenberg committed:
Update references.bib

Tom-van-Woudenberg committed:
Update gof.md

fmeer commented:
I would prefer to have Robert and Sandra still included as authors of the overall book. Especially when we do not emphasize "editors". What happened to the "editors" by the way? I liked that one, to avoid the impression that we are claiming authorship of the whole book which is in fact a collection of chapters with many different authors.

Tom-van-Woudenberg committed:
fix dois?

Tom-van-Woudenberg committed:
Merge branch 'credits-2025' of https://github.com/TUDelft-MUDE/book into credits-2025

Tom-van-Woudenberg committed:
make editors more clear

Tom-van-Woudenberg commented:
> I would prefer to have Robert and Sandra still included as authors of the overall book. Especially when we do not emphasize "editors". What happened to the "editors" by the way? I liked that one, to avoid the impression that we are claiming authorship of the whole book which is in fact a collection of chapters with many different authors.

Robert, Sandra (and me) were only intended as editors of the book, although the 'Eds.' was only mentioned when referring to individual chapters.

It's really not my intention to claim authorship. But if we want to have as the main names the editors, we shouldn't keep old the editors there. That will then grow to an ever growing list. How about if I add editor more clearly? As implemented in b122d7b9a6e43536a5eac51e0fbb8dff9e9ee675

Tom-van-Woudenberg commented:
>  As implemented in [b122d7b](https://github.com/TUDelft-MUDE/book/commit/b122d7b9a6e43536a5eac51e0fbb8dff9e9ee675)

Hmm, that leads to references of:

> None to claim their bones (n.d.). Modelling, Uncertainty and Data for Engineers (MUDE) Textbook. Delft University of Technology. https://mude.citg.tudelft.nl/book CC BY 4.0.
> 
> None to claim their bones (2024). Modelling, Uncertainty and Data for Engineers (MUDE) Textbook. Delft University of Technology. https://mude.citg.tudelft.nl/book/2024 CC BY 4.0. [doi:10.5281/zenodo.16236358](https://doi.org/10.5281/zenodo.16236358)

Add all the authors? At some point @rlanzafame and @SandraVerhagen  decided it makes sense to have the editors there because that's common for books in science

fmeer commented:
> It's really not my intention to claim authorship.

I know and the encouragement to refer to individual chapters/pages should make our intentions clear to whoever reads carefully, but still good to be cautious. Adding (Eds) anyway helps for that and then it does makes sense to limit the list to active editors. At the same time, the book that we have now is still largely the book for which Robert was the main editor. @rlanzafame any thoughts on this?

fmeer commented:
How about "author = Delft University of Technology"?

Tom-van-Woudenberg committed:
simplify

Tom-van-Woudenberg committed:
TU Delft as author

Tom-van-Woudenberg committed:
fix bibtex

Tom-van-Woudenberg committed:
small fixes

Tom-van-Woudenberg commented:
> How about "author = Delft University of Technology"?

That might work, but then we deviate even more from our recommendation:

van Woudenberg, T., van der Meer, F., Ding, J. (Eds.) (n.d.), Modelling, Uncertainty and Data for Engineers (MUDE) Textbook, Delft University of Technology. https://mude.citg.tudelft.nl/book, CC BY 4.0.

-->

Delft University of Technology (n.d.). Modelling, Uncertainty and Data for Engineers (MUDE) Textbook. https://mude.citg.tudelft.nl/book. CC BY 4.0.

I think having the full list of authors and editors there (indicating who is editor) and keep that up-to-date every year would be the best solution

Tom-van-Woudenberg committed:
replace edtiors with full list of authors

Tom-van-Woudenberg commented:
> I think having the full list of authors and editors there (indicating who is editor) and keep that up-to-date every year would be the best solution

Proposed in https://github.com/TUDelft-MUDE/book/tree/full-authors

fmeer committed:
Bibtex alternative

fmeer commented:
"book" instead of "misc" helps to get the editors to show up with bibtex

Tom-van-Woudenberg committed:
book bib in references too

Tom-van-Woudenberg committed:
revert to misc

Tom-van-Woudenberg commented:
> "book" instead of "misc" helps to get the editors to show up with bibtex

This has other issues, URL is not shown and publisher required.

I think we have to make two choices:
1. Editors in the citation (https://mude.citg.tudelft.nl/book/credits-2025/credits.html) v.s. all authors with editors noted (https://mude.citg.tudelft.nl/book/full-authors/credits.html) (I've sorted authors / editors alphabetically)
2. Editors as current MUDE MT v.s. keep editors that are not active anymore

My preference would be:
1. all authors
2. only current MUDE MT as editors

Tom-van-Woudenberg committed:
Update credits.md

fmeer commented:
> This has other issues, URL is not shown and publisher required.

When I compile the "book" reference with bibtex, the URL under "note = " does show in the final result and the missing publisher is only a warning.

Tom-van-Woudenberg committed:
Update credits.md

Tom-van-Woudenberg committed:
Update references.bib

Tom-van-Woudenberg committed:
Update references.bib

Tom-van-Woudenberg committed:
Update credits.md

Tom-van-Woudenberg commented:
> When I compile the "book" reference with bibtex, the URL under "note = " does show in the final result and the missing publisher is only a warning.

Ah yes, that's correct leads to this: https://mude.citg.tudelft.nl/book/book_iso_misc/credits.html.

Still, the questions are:

> * Editors in the citation (https://mude.citg.tudelft.nl/book/credits-2025/credits.html) v.s. all authors with editors noted (https://mude.citg.tudelft.nl/book/full-authors/credits.html) (I've sorted authors / editors alphabetically)
> * Editors as current MUDE MT v.s. keep editors that are not active anymore

rlanzafame commented:
My opinion, in short: 1) keep the citation, editors and authors as-is, adding names as needed based on merit; 2) you are trying to modify the _book_ to reflect intricacies of the _module._ I'll try to explain. If you agree/disagree, maybe the final result of the conversations in this PR can be summarized in a README or on the MUDE page.

Now the long version.

Probably good to state that in my opinion the MUDE Textbook is distinct from the module itself; I see it like a paper textbook, though it is obviously different in many ways. Key thing here is that I believe we have built something that will last for many years in a form similar to what it is now; it is not a book that is made "new" every year. In that the digital aspect and source code is one of the things making the book different from a paper book, it is important to preserve it digitally; this is why there are 2 zenodo records: one to preserve the source code while listing all authors in a relatively easy automatic way, the other to display the book and citation in a more traditional way, so that you can easily acknowledge contributions as needed (noting that Zenodo is really not meant to be the primary place to read this anyway; that should be the Credits page). Perhaps good to also note that I think the Credits page is mostly targeting non-student readers! (students should read it, but it's irrelevant with regards to the study material)

I feel very strongly that the _citation_ should remain as-is because it reflects who actually did the work to create the book in different ways (e.g., creative contributions, editing, code, etc). Think of this like the typical TUD lecture notes: someone writes something, another person adds, people retire, etc...typically the names stay on the document until someone decides the list is too long or the format has changed significantly from the original. Either way, the original people should still be acknowledged. The reason Tom, Sandra and I are editors is that in my opinion each of those people had a significant impact on broad areas of the book. This is to distinguish (but not _diminish_!) the excellent work of others who primarily worked on the content in specific chapters. Depending on the contribution, this is the authors or acknowledgements in a chapter. I don't see any reason to keep the list of editors or authors short, as it reflects peoples contributions. Tom, I know you hate claiming credit for your work, but recognize that you being an editor illustrates clearly it was not only me making the book, and that you indeed played a huge role in creating the book.

The proposal to list Jialei, Frans and Tom as editors or authors for this year is, I believe, misdirected: I think you should very much be acknowledged as MUDE MT for the work you are currently doing, and will do, but that doesn't guarantee a direct contribution to the book. Maybe what could be done here on the Credits page (and perhaps also the home page) is to add  an acknowledgement and explanation. A short sentence or two could suffice, also with a hyperlink to the main MUDE website (e.g., a MUDE MT or MUDE Team explanation).

I fully expect Frans and Jialei to become editors at some point. Perhaps others this year, as well, if people really go beyond their "own" chapter. _When_ each of you becomes an editor probably depends on what your focuses and activities are this year. I would say that approving PR's and checking that the book doesn't "break  is _not_ editor-worthy, whereas working with teachers to add, improve, create content in a way that improves and aligns with the greater book: yes. I trust you all to determine if you have done this and add yourselves, or others. If in doubt, Tom, Sandra and I could discuss.

Important to note explicitly: obviously we did a _lot_ of work getting the book set up, but I don't think that should set the bar in terms of what others have to do to become an editor. Major changes aren't needed; stewardship of the content, format and vision is essential for maintaining and improving the book, and this should be acknowledged by being listed as an editor. The same holds within an individual chapter.

To be honest, my preference would be to add editors and authors _after_ contributions are made, not in advance (doing it within a PR where a lot of content is added would be fine!). Credits page could be updated during the year (you can add a note about the list of authors potentially updating if you want) and finalized in Feb or so. One reason for this is obvious: giving people credit where credit is due. Another reason is very MUDE-specific: the team is complicated and attrition is high. You've seen plenty of people come and go, so for certain chapters I think you can see that the omission of a name can be equally as important as its inclusion...

Now that my opinions are stated, there's a bit of the technical/practical side: I hate messing with bibtex entries, which is why i tried to explicitly state what the citation should look like using markdown on the Credits page and the Zenodo pages. I did confirm that the bibtex code block i added generated the right reference and citation in the MUDE book, though. This is also partly why I made the second manual Zenodo entry: to make sure that there was a record/DOI that could always be customized the way we want it, regardless of how Zenodo or cff files changed over time. In other words: I think some of the conversation above is about the way a bibtex entry is formed, which I think should be secondary (and maybe this can be spun off into other issues, if you can't get it into the right format yet).

In closing: sorry for the length, and strong opinions. As you know I am passionate about MUDE and am sad that I had to leave TU Delft, so perhaps that makes me biased in these opinions. Nevertheless, I do think we have something special, and I would really love to see that in 10 years the editor and author list is HUGE and that as people come and go we can all look back on that and be proud of the unique things we did and the special club we are part of :)

Tom-van-Woudenberg committed:
adapt to comments Robert

Tom-van-Woudenberg committed:
typo

Tom-van-Woudenberg committed:
fix bib

Tom-van-Woudenberg committed:
final?

Tom-van-Woudenberg commented:
Thanks for your extensive reply @rlanzafame!

> I feel very strongly that the _citation_ should remain as-is because it reflects who actually did the work to create the book in different ways (e.g., creative contributions, editing, code, etc). Key thing here is that I believe we have built something that will last for many years in a form similar to what it is now; it is not a book that is made "new" every year.

You convinced me on the 'it is not a book that is made "new" every year".

> The proposal to list Jialei, Frans and Tom as editors or authors for this year is, I believe, misdirected: I think you should very much be acknowledged as MUDE MT for the work you are currently doing, and will do, but that doesn't guarantee a direct contribution to the book. Maybe what could be done here on the Credits page (and perhaps also the home page) is to add an acknowledgement and explanation. A short sentence or two could suffice, also with a hyperlink to the main MUDE website (e.g., a MUDE MT or MUDE Team explanation).

I added a comment on this here: https://github.com/TUDelft-MUDE/book/pull/107/commits/a91958d4a3a383a7335305e9c4a385071c519632#diff-502c982a7b1d0914e715994edfa7efc1189d7fc6d8be22a915c01eb2f1359a58R69

> I trust you all to determine if you have done this and add yourselves, or others. If in doubt, Tom, Sandra and I could discuss.

I still feel some discomfort about how to manage the list of 'editors' in the far future, but for now this might be the best strategy.

> To be honest, my preference would be to add editors and authors _after_ contributions are made, not in advance (doing it within a PR where a lot of content is added would be fine!). Credits page could be updated during the year (you can add a note about the list of authors potentially updating if you want) and finalized in Feb or so.

I've added a note on this here: https://github.com/TUDelft-MUDE/book/pull/107/commits/a91958d4a3a383a7335305e9c4a385071c519632#diff-502c982a7b1d0914e715994edfa7efc1189d7fc6d8be22a915c01eb2f1359a58R98

> Now that my opinions are stated, there's a bit of the technical/practical side: I hate messing with bibtex entries, which is why i tried to explicitly state what the citation should look like using markdown on the Credits page and the Zenodo pages. I did confirm that the bibtex code block i added generated the right reference and citation in the MUDE book, though. This is also partly why I made the second manual Zenodo entry: to make sure that there was a record/DOI that could always be customized the way we want it, regardless of how Zenodo or cff files changed over time. In other words: I think some of the conversation above is about the way a bibtex entry is formed, which I think should be secondary (and maybe this can be spun off into other issues, if you can't get it into the right format yet).

Yes, it's a secondary issue. But I think the book format which prioritizes editors (as suggested by @fmeer) is more clear, and the version can be included too. I've updated it now

@fmeer, any opinions from your side related to all this?
@rlanzafame, any opinions on the other changes to the credits page, mainly restructuring of first part under `# Credits and license`

fmeer commented:
I agree that Robert and Sandra still deserve to be there prominently, as was my first reaction to this PR as well. I do also Jialei and I are taking responsibility for the contents of MUDE, which includes the book, and we are in fact acting as editors now. Our contribution will remain different from those of the original creators, we can decide by the time we release the final 2025 book, looking back on our actual contribution, but I think we should be included as editors then.

Yes, the book is separate from MUDE, we keep module logistics out of the book because the book is not just written for our own present students. For instance, I would not mind removing **Q1 topics** and **Q2 topics** at some point, and releasing the whole book at once also to our students, to make it even more like a classical text book. At the same time, the book is and remains a living document that is primarily tailored to the module, where the MUDE team is in the lead. If the contents of MUDE change over time, the book will be updated accordingly. In this sense, the book is not that separate from MUDE. 

This discussion is a good exercise, because I imagine similar issues will come up about authorship of individual pages. What is even more important than proper crediting is that we should incentivize improvements to the book. Maybe this is personal and not the same for everyone, but when I am listed as editor, I will feel more responsible, and more free as well, to make changes where I see potential for improvement. The same holds for individual pages. Having someone else's name as author creates a barrier for making changes. I do not have a solution to that, but this discussion does remind me that we need to encourage our teachers to contribute and continue to be sharp and generous on updating credits. What to do with legacy contributors (i.e. when to remove names) may become tricky.

rlanzafame commented:
@Tom-van-Woudenberg 

Looks good!

As for this:

> I still feel some discomfort about how to manage the list of 'editors' in the far future, but for now this might be the best strategy.

Don't worry, I think if you just let it be merit-based it will be more obvious than you think!

rlanzafame commented:
@fmeer 

Excellent points. I agree.

Especially the incentives are important (and a recognized challenge in general, not just MUDE). Maybe one thing to consider for this would be to note who is actively maintaining the content for the current year on the front page of each chapter. For example, "First Last and First Last are responsible for the content in this Chapter, which is written by Last, First et al. "Find out more here."

Presumably the active content person would be listed in the author list anyway, but more important it means that this person is identified clearly and prominently to students and anyone else reading the content, not only the citation.

Tom-van-Woudenberg commented:
I think we have enough to merge this PR.

> Especially the incentives are important (and a recognized challenge in general, not just MUDE). Maybe one thing to consider for this would be to note who is actively maintaining the content for the current year on the front page of each chapter. For example, "First Last and First Last are responsible for the content in this Chapter, which is written by Last, First et al. "Find out more here."

This sounds a bit like 'editors' on topic-level ( = content coordinators). I wouldn't know whether that really lowers the barrier for contributions.

Tom-van-Woudenberg committed:
Merge branch '2025-draft' into credits-2025
